recipe/schemas.py

```diff
@@ -60,6 +60,26 @@ def _make_sqlalchemy_datatype_lookup():
     "age": lambda fld: func.date_part("year", func.age(fld)),
     "none": lambda fld: fld,
     None: lambda fld: fld,
+    # Percentile aggregations do not work in all engines
+    "median": func.median,
+    "percentile1": lambda fld: func.percentile_cont(0.01).within_group(fld),
```

Comment:
Instead of hardcoding all of these things, can we just have a syntax like:

```yaml
aggregation:
    percentile: 75
```

?

Reply:
@radix Theoretically* yes, but I don't think it's worth the complexity.

Aggregations go thru this code https://github.com/juiceinc/recipe/blob/master/recipe/shelf.py#L258

Two thoughts:

1) I want to support a `field: "{aggregation}({column})"` syntax. This would be a 

2) post-validated shelves need to be simple python objects because we store them in DataSource. Given this the logic of decoding percentage would have to be here https://github.com/juiceinc/recipe/blob/master/recipe/shelf.py#L258 and that's spreading out where things happen more than I'd like.